core/src/main/java/de/danoeh/antennapod/core/feed/FeedItem.java

```diff
@@ -388,6 +391,22 @@ public boolean hasChapters() {
         return hasChapters;
     }
 
+    public void setAutoDownload(boolean autoDownload) {
+        this.autoDownload = autoDownload;
+    }
+
+    public boolean getAutoDownload() {
+        return this.autoDownload;
+    }
+
+    public boolean isAutoDownloadable() {
+        return this.hasMedia() &&
+                false == this.getMedia().isPlaying() &&
```

Comment:
Nice, there even is a name for that: http://en.wikipedia.org/wiki/Yoda_conditions
I wonder, if I can call myself a code jedi, now